x/stakeibc/keeper/icacallbacks_redemption.go

```diff
@@ -91,6 +91,7 @@ func RedemptionCallback(k Keeper, ctx sdk.Context, packet channeltypes.Packet, t
 			}
 			userRedemptionRecord.IsClaimable = true
 			k.RecordsKeeper.SetUserRedemptionRecord(ctx, userRedemptionRecord)
+			k.Logger(ctx).Info(fmt.Sprintf("[REDEMPTION] completed on %s", hostZoneId))
```

Comment:
this loop is over `userRedemptionRecords`, I think we want this log line at the end of the function?

Reply:
good catch. done